filebeat/module/elasticsearch/gc/_meta/fields.yml

```diff
@@ -0,0 +1,16 @@
+- name: gc
+  type: group
+  description: >
+  fields:
+    - name: relative_process_timestamp_secs
```

Comment:
These should adhere to our naming conventions: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html We can also tackle this in a follow up PR.

Reply:
Sure, we can fix on a follow up PR.